classes.js

```diff
@@ -0,0 +1,54 @@
+class Human {
+  constructor(name, age){
+   this.name = name;
+   this.age = age;
+
+  }
+  ageOneYear () {
+  return this.age++;
+  }
+
+  eating () {
+    return "mmm, mmm, mmm, I'm love'n it";
+  }
+}
+
+let koohyar = new Human("Koohyar", 28);
+ console.log(koohyar.age);
+
+koohyar.ageOneYear();
+console.log(koohyar.age);
+koohyar.eating()
+console.log(koohyar.eating());
+
```

Comment:
Small note: variable names that are classes should begin with an uppercase.
Great job with this assignment!